runbotics/runbotics-desktop/src/action/microsoft/excel/excel.service.ts

```diff
@@ -139,14 +140,29 @@ export class ExcelService {
 
     /**
      * @see https://learn.microsoft.com/en-us/graph/api/range-update?view=graph-rest-1.0&tabs=javascript
+     * @description {string} url requires range address in a format of column letters, e.g. "A1:C3"
      */
-    setRange(session: ExcelSession, address: string, values: string | ExcelCellValue[][]) {
-        const url = `/worksheets/${session.worksheetName}/range(address='${address}')`;
-
+    setCells(session: ExcelSession, startCell: string, values: ExcelCellValue[][]) {
         if (!Array.isArray(values)) {
             values = JSON.parse(values) as ExcelCellValue[][];
         }
 
+        if (!values.length) {
+            throw new Error(CloudExcelErrorMessage.setCellsIncorrectInput());
+        }
+        
+        const startColumnLetter = startCell.match(/[A-Z]+/).toString();
```

Comment:
there's (or soon will be) 

export enum ActionRegex {
 EXCEL_COLUMN_NAME = '^[a-zA-Z]+$', // add/use this line
}

Reply:
imo we could even create ExcelRegex and keep there all excel patterns